userguide/platform/includes/kaui-configuration-properties-table.adoc

```diff
@@ -48,17 +48,17 @@ KILLBILL_URL/KAUI_KILLBILL_URL   |Specifies the Kill Bill URL   |http://127.0.0.
 
 KILLBILL_DISABLE_SSL_VERIFICATION/
 
-KAUI_KILLBILL_DISABLE_SSL_VERIFICATION   |Coming Soon   |Coming Soon
+KAUI_KILLBILL_DISABLE_SSL_VERIFICATION   |Whether to ignore SSL certificates checks   |false
 |kaui.read_timeout/
 
 KILLBILL_READ_TIMEOUT/
 
-KAUI_KILLBILL_READ_TIMEOUT   |Coming Soon   |60000
+KAUI_KILLBILL_READ_TIMEOUT   |Specifies the read timeout   |60000
 |kaui.connection_timeout/
 
 KILLBILL_CONNECTION_TIMEOUT/
 
-KAUI_KILLBILL_CONNECTION_TIMEOUT   |Coming Soon   |60000
+KAUI_KILLBILL_CONNECTION_TIMEOUT   |Specifies the connection timeout   |60000
```

Comment:
Same comment as above, we need to confirm if this is the KB connection timeout and if so elaborate this description